src/pages/applications.js

```diff
@@ -343,6 +344,17 @@ const emergingProjects = [
       {label: 'Website', url: 'https://www.loxilb.io/'},
     ],
   },
+  {
+    logoUrl: 'https://github.com/sustainable-computing-io/kepler',
+    name: 'Kepler',
+    logo: keplerLogo,
+    title: 'Kubernetes-based Efficient Power Level Exporter',
+    description: `Kepler (Kubernetes-based Efficient Power Level Exporter) uses eBPF to probe energy related system stats and exports as Prometheus metrics.`,
```

Comment:
Thanks a lot for adding Kepler. Could you write a slightly bigger description so it aligns in terms of verbosity roughly with most other projects listed in the application landscape?

Would also be great if you could rebase given there's a merge conflict. Thanks a lot!